common/persistence/sql/sqlClusterMetadataManager.go

```diff
@@ -97,10 +96,13 @@ func (s *sqlClusterMetadataManager) GetImmutableClusterMetadata() (*p.InternalGe
 }
 
 func (s *sqlClusterMetadataManager) GetClusterMembers(request *p.GetClusterMembersRequest) (*p.GetClusterMembersResponse, error) {
-	pageToken := uint64(0)
-	if len(request.NextPageToken) > 0 {
-		pageToken = binary.LittleEndian.Uint64(request.NextPageToken)
+	var lastSeenHostId []byte
+	if len(request.NextPageToken) == 16 {
+		lastSeenHostId = request.NextPageToken
+	} else if len(request.NextPageToken) > 0 {
```

Comment:
is there a test case that exercises this else if error branch?

Reply:
No, this is a follow-up that will be addressed with some more work done around UUID parsing. This should really never occur though given the database size is 16 bytes and so is our UUID type, so it would require client error or database misconfig/corruption.